app/retail/views.py

```diff
@@ -39,6 +39,16 @@
 from .utils import build_stat_results, programming_languages
 
 
+def get_activities(tech_stack=None, num_activities=15):
```

Comment:
❤️

Reply:
only 5 are shown at a time on the landing page, though the user can scroll them because the `max-height: 500px; overflow-y: scroll` property of that div